lib/appsignal/transaction.rb

```diff
@@ -156,6 +158,29 @@ def set_tags(given_tags = {})
       @tags.merge!(given_tags)
     end
 
+    # Add breadcrumbs to the transaction.
+    #
+    # @param category [String] category of breadcrumb
+    #   e.g. "UI", "Network", "Navigation", "Console".
+    # @option action [String] name of breadcrumb
+    #   e.g "The user clicked a button", "HTTP 500 from http://blablabla.com"
+    # @option metadata [Hash<String,String>]  key/value metadata in <string, string> format
+    # @option time [Time] time of breadcrumb, should respond to `.to_i` defaults to `Time.now.utc`
+    # @return [void]
+    #
+    # @see Appsignal.add_breadcrumb
```

Comment:
I don't see this helper method? Do we need to add it to `Appsignal::Helpers::Instrumentation`? The docs are for the method on the Transaction class.